### DISCUSSION TITLE
where is the "@iorsan/node-red-contrib-firebase-notification"?

### DISCUSSION THREAD

**Discussion by rimmomo** [ismaelOrtega14/node-red-contrib-firebase-cloud-messaging#20] [Q&A]
where is the "@iorsan/node-red-contrib-firebase-notification"?
And, do I just put the key value in the key file?
The key path fails even if i typ "/home/mydir/xx.key".
How can I set it up?
I'll be waiting for your reply.
Thank you.

**Comment by ismaelOrtega14** [comment]
Hello,
Did you close this?
Did you figure it out?
